clang-tools-extra/test/clang-doc/basic-project.mustache.test

```diff
@@ -60,6 +60,17 @@ HTML-SHAPE:         <div class="content">
 HTML-SHAPE:             <section class="hero section-container">
 HTML-SHAPE:                 <div class="hero__title">
 HTML-SHAPE:                     <h1 class="hero__title-large">class Shape</h1>
+HTML-SHAPE:                    <div class="hero__subtitle">
+HTML-SHAPE:                                    <div>
+HTML-SHAPE:                                        <p> Abstract base class for shapes.</p>
+HTML-SHAPE:                                    </div>
+HTML-SHAPE:                                        <div>
+HTML-SHAPE:                                    <p></p>
+HTML-SHAPE:                                </div>
+HTML-SHAPE:                                    <div>
+HTML-SHAPE:                                    <p> Provides a common interface for different types of shapes.</p>
+HTML-SHAPE:                                </div>
+HTML-SHAPE:                                                </div>
```

Comment:
Do we have any notion about why the format w/ the tags is so wonky? The template looks fine. Are we handling whitespace incorrectly in the mustache lib? or do we need to format the template differently to get these to line up correctly?

Reply:
Can you file a bug about this and CC me? you can tag it w/ clang-doc for now, even though its an issue w/ mustache lib. I can try to fix that maybe next week.